src/httpserver.cpp

```diff
@@ -13,6 +13,7 @@
 #include "ui_interface.h"
 #include "util.h"
 
+#include <mutex>
```

Comment:
is seems unrelated. why we need it?

Reply:
windows build was giving me an error std::mutex undefined.  IDK why travis was not seeing it.